server/src/main/java/org/candlepin/util/X509Util.java

```diff
@@ -230,7 +230,20 @@ public String createFullContentPath(String contentPrefix, ProductContent pc) {
         return contentArches;
     }
 
+    /**
+     * Returns true/false depending on whether or not there is a match/mismatch between the consumer's supported
+     * architectures and the content's supported architectures.
+     *
+     * @param consumerArches The architectures that the consumer supports
+     * @param contentArches The architectures that the content supports
+     * @return False if none of the content's architectures match to any of the consumer archituctures
```

Comment:
Typo with the second "architectures" in this line, and the comma should move up (or if it's too long, arch moves down)

Reply:
Done. also added a unit test